source/WebApp/package.json

```diff
@@ -106,7 +106,7 @@
     "dateformat": "1.0.12",
     "lz-string": "1.4.4",
     "marked": "0.7.0",
-    "mirrorsharp": "file:../#external/mirrorsharp/WebAssets/dist",
+    "mirrorsharp": "file:../%23external/mirrorsharp/WebAssets/dist",

```

Comment:
Not fully sure about this one -- I think I tried this before and run into some problems.
I'll need to try it out again before merging.